homeassistant/components/androidtv_remote/strings.json

```diff
@@ -54,5 +54,17 @@
         }
       }
     }
+  },
+  "services": {
+    "send_text": {
+      "name": "Send text",
+      "description": "Sends text to an Android device.",
+      "fields": {
+        "text": {
+          "name": "Text",
+          "description": "Text to 'type'"
```

Comment:
what does `'type'` mean? Show on the screen? Send it as input? I think we can be more descriptive